cmd/beekeeper/cmd/check.go

```diff
@@ -47,8 +50,10 @@ func (c *command) initCheckCmd() (err error) {
 	cmd.PersistentFlags().BoolVar(&insecureTLSDebugAPI, optionNameDebugAPIInsecureTLS, false, "skips TLS verification for debug API")
 	cmd.PersistentFlags().BoolVar(&disableNamespace, optionNameDisableNamespace, false, "disable Kubernetes namespace")
 	cmd.PersistentFlags().Bool(optionNameInsecureTLS, false, "skips TLS verification for both API and debug API")
-	cmd.PersistentFlags().StringP(optionNameNamespace, "n", "", "Kubernetes namespace, must be set or disabled")
+	cmd.PersistentFlags().StringP(optionNameNamespace, "n", "bee", "Kubernetes namespace, must be set or disabled")
```

Comment:
Is this dangerous, as it is defaulting to the public cluster namespace?

Reply:
@janos have some other suggestion? `default`?

Reply:
Why not to leave it blank as it is now?